.travis.yml

```diff
@@ -1,20 +1,22 @@
-language: c
+language: ruby
```

Comment:
@reitermarkus Would rather avoid this if possible; the boot time is slower. Perhaps `rvm: system` is sufficient?

Reply:
`language: ruby` seems to be required to be able to specify `rvm: system`